src/main/resources/application.yaml

```diff
@@ -25,6 +25,10 @@ spring:
   output:
     ansi:
       enabled: detect
+  datasource:
```

Comment:
anyone who will use this config have to create an application.local.yaml file with own creds and etc for local dev